node/cn/channel_manager_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package cn
+
+import (
+	"github.com/klaytn/klaytn/consensus/istanbul/backend"
+	"github.com/klaytn/klaytn/networks/p2p"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// TestChannelManager tests registering and retrieving of message channels.
+func TestChannelManager(t *testing.T) {
+	cm := NewChannelManager(1)
```

Comment:
Multichannel is not being tested, do you plan to add it?

Reply:
I will update a one in this PR. Thanks for the review =)